Amplify/Categories/Geo/GeoCategoryTypes.swift

```diff
@@ -0,0 +1,141 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import CoreLocation
+
+/// Results handler for Amplify Geo.
+public typealias GeoResultsHandler<T> = (Result<T, Error>) -> Void
+
+/// A pair of coordinates to represent a location (point).
+public struct Coordinates {
+    /// The latitude of the location.
+    public let latitude: Double
+    /// The longitude of the location.
+    public let longitude: Double
+
+    /// Initializer
+    public init(latitude: Double, longitude: Double) {
+        self.latitude = latitude
+        self.longitude = longitude
+    }
+}
+
+public extension Coordinates {
+    /// Initialize a Location from a CLLocationCoordinate2D
+    /// - Parameter location: The CLLocationCoordinate2D to use to initialize the
+    /// Location.
+    init(_ coordinates: CLLocationCoordinate2D) {
+        self.init(latitude: coordinates.latitude, longitude: coordinates.longitude)
+    }
+}
+
+public extension CLLocationCoordinate2D {
+    /// Initialize a Location from a CLLocationCoordinate2D
+    /// - Parameter location: The CLLocationCoordinate2D to use to initialize the
+    /// Location.
+    init(_ coordinates: Coordinates) {
+        self.init(latitude: coordinates.latitude, longitude: coordinates.longitude)
+    }
+}
+
+/// A bounding box defined by southwest and northeast corners.
+public struct BoundingBox {
+    /// The southwest corner of the bounding box.
+    public let southwest: Coordinates
+    /// The northeast corner of the bounding box.
+    public let northeast: Coordinates
+
+    /// Initializer
+    public init(southwest: Coordinates, northeast: Coordinates) {
+        self.southwest = southwest
+        self.northeast = northeast
+    }
+}
+
+/// The area to search.
+public enum SearchArea {
+    /// Searches for results closest to the given coordinates.
+    case near(Coordinates)
+    /// Filters the results by returning only Places within the provided bounding box.
+    case within(BoundingBox)
+}
+
+public extension SearchArea {
+    /// Creates a SearchArea that returns results closest to the given
+    /// CLLocationCoordinate2D.
+    /// - Parameter coordinates: The coordinates for the search area.
+    /// - Returns: The SearchArea.
+    static func near(_ coordinates: CLLocationCoordinate2D) -> SearchArea {
+        .near(Coordinates(coordinates))
+    }
+}
+
+/// A place defined by a label, location (point), and optional additional locality
```

Comment:
`A place defined by a label` - I see that only the coordinates are non-optional.

Reply:
right, this recently changed based on what we get back from the location service and I updated the struct but no the doc.  good catch!  updated.